lib/bolt/executor.rb

```diff
@@ -134,6 +134,18 @@ def log_action(description, targets)
       results
     end
 
+    def log_start_plan(plan_name)
```

Comment:
A pattern we used in `log_action` was to take a block and do the logging around yielding to that block. You could still use an `ensure` section to make sure the finished message is logged.

I'd be in favor of that here. And if need be splitting some of the run_plan function out into other functions.

Reply:
Otherwise this seems to follow the semantics we've defined for other logging, so 👍

Reply:
Great idea!

I saw the other functions `start_plan` and `finish_plan` and flipped the coin.  I'll change to block semantics and resubmit. 

Thanks for the feedback!

Reply:
Yeah, those could've followed a similar pattern. I may fix that up at some point.